tools/xcms_summary/abims_xcms_summary.xml

```diff
@@ -1,4 +1,4 @@
-<tool id="abims_xcms_summary" name="xcms process history" version="@TOOL_VERSION@+galaxy0">
+<tool id="abims_xcms_summary" name="xcms process history" version="@TOOL_VERSION@+galaxy1">
```

Comment:
This does not need to be bumped as you increase the TOOL_VERSION already.

Reply:
OUhai, it was really quick!

CAMERA isn't the main tool of this wrapper. It's XCMS.
Indeed, it can be confusing :/

Reply:
ah, I see ... Ok, sorry.